### PR TITLE
Nitrogen doesn't heal toxin damage for Vox anymore.

### PR DESCRIPTION
**Pull request by Irradiation**
~~Because ayy lmao, Oxygen deals 0.5 toxins but Nitrogen heals 1 toxins.~~ Fugging vox creep. Just slurp Antitox, Peptobismol, Tricord, DD, GRAVY, literally anything else.

EDIT:  Nitrogen doesn't heal toxin damage anymore for birds..

[balance][tweak][consistency]

:cl:
 * tweak: The Nitrogen reagent doesn't heal toxin damage for Vox anymore.

### CODE REVIEW

**Comment by SonixApache** [comment]
you were a little too fast; i was going to suggest tying it to the actual lungs a human mob has instead of just saying 'lol vox'

**Comment by Irradiation** [comment]
Hayai.

**Comment by SonixApache** [comment]
Another alternative would be creating Nitralin which does what dexalin does but for vox lungs, move this there, and make dexalin toxic to vox.

**Comment by ShiftyRail** [comment]
Sonix, this would make medbay a complete nightmare

**Comment by SonixApache** [comment]
I don't think having to give someone a different pill would make it a complete nightmare. Unless you're putting dexalin in the mix for some reason.
Add it to the quick-guide on the wiki and it'll be made every shift.

Also proof that PJB had some good ideas after all.

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -1125,7 +1125,7 @@
 
 	if(alien && alien == IS_VOX)
 		M.adjustOxyLoss(-2 * REM)
-		M.adjustToxLoss(-2 * REM)
+		M.adjustToxLoss(REM)
```
Comment (Exxion) [inline code/modules/reagents/Chemistry-Reagents.dm]: I think you forgot a `-` there
This makes it *deal* toxin damage

**Comment by ShiftyRail** [comment]
I put Dexalin+ in the mix and I think it's a common filler

File: code/modules/reagents/Chemistry-Reagents.dm (hunk 1 above)
Comment (Irradiation) [reply code/modules/reagents/Chemistry-Reagents.dm]: "Oops"

**Comment by bathosbathtime** [comment]
For better or worse, making dex+ toxic to vox will result in a lot of dead vox.

**Comment by ShiftyRail** [comment]
Le ven. 25 janv. 2019 5:38 PM, Bathos Bathtime <notifications@github.com> a
écrit :

**Comment by SGuivenne** [comment]
I support this measure only if medbay actively killing vox via cryo is not considered job ban worthy.  Which should be the case anyway as it does the station a service.

**Comment by Irradiation** [comment]
EDIT: no more toxin healing for birbs.

**Comment by Exxion** [comment]
Open a new PR for that
Changing the basis of a PR makes everything complicated

**Comment by Irradiation** [comment]
Okay

**Pull request closed without merging**